PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,18 @@
 #### What does this PR do?
+it adds route to edit the name of a specific political party
+it add unit-tests for the API endpoints
+
 #### Description of Task to be completed?
+* Have the following api endpoints working
+```POST /api/v1/party/```
+```GET /api/v1/party```
+```GET/api/v1/party/:id```
+```PATCH/api/v1/party/:id/name```
+
 #### How should this be manually tested?
-#### Any background context you want to provide?
-#### What are the relevant pivotal tracker stories?
-#### Screenshots (if appropriate)
-#### Questions:
\ No newline at end of file
+*Clone this repository and run ```npm install``` followed by ```npm start``` to start the server*
+* Use Postman to test the endpoints above and use the following header
+*key*: Content-type
+*value*: Application/json
+
+#### What are the relevant pivotal tracker stories?
```

Comment:
This file shows you did not branch out of develop to create this branch it hs the update of this branch `ft-delete-party-api-163446193` you are creating office here but have party as your PR description